torchci/pages/api/drci/drci.ts

```diff
@@ -0,0 +1,92 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { getOctokit } from "lib/github";
+import { Octokit } from "octokit";
+import fetchRecentWorkflows from "lib/fetchRecentWorkflows";
+import { RecentWorkflowsData } from "lib/types";
+
+const NUM_MINUTES = 15;
```

Comment:
should be consistent with top level variables. We prefer using ALL_CAPS_WITH_UNDERSCORES over camelCase